common/test/acceptance/tests/video/test_video_times.py

```diff
@@ -160,28 +160,3 @@ def test_video_end_time_with_seek(self):
         self.video.wait_for_state('pause')
 
         self.assertIn(self.video.position, ('0:35', '0:36'))
-
-    @flaky  # TODO fix this, see TNL-1619
-    def test_video_finish_time_with_seek(self):
```

Comment:
This tests functionality that no longer exists. More specifically, this test makes sure that if the user seeks past the instructor-specified end time, the video plays until the end of the video. However, with this PR, the video will stop playing at the instructor-specified end time.